Kf update tables

katherinevfry opened the pull request:
## Description
When a user assigns a table to a reservation, they are now only able to choose from tables that are available (i.e., not assigned to another reservation). The Seating view now shows which tables are available and which are reserved. If a user updates what table is assigned to a reservation, the previous table becomes available again and the new table is listed as reserved.

## Related Issue

## Motivation and Context
Previously, on the Seating view, the listed availability of a table was completely arbitrary. It now reflects the actual data.

## How Can This Be Tested?
git fetch
git checkout kf-update-tables

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

katherinevfry committed:
stuff

katherinevfry committed:
stashing my stuff

katherinevfry committed:
this commit has some issues

katherinevfry committed:
working update availability

katherinevfry committed:
fully functional table reservations

katherinevfry committed:
merged in development

jrobinson0529 reviewed (commented):
On seating view there is a 'Table undefined' Doesn't look like a big issue but if it is not a pain to fix we probably should rid of it. Could be faulty data im not sure just at first glance

https://i.gyazo.com/9635e09b7143a0f8b3c3d3c1132e236c.png

katherinevfry committed:
import statement for landing page

katherinevfry commented:
- I added the import statement for the logo to the landing page
- The bug on the seating page won't replicate. It may have just been a data issue. I fixed the data!

chiestroud reviewed (approved):
👍
